notebooks/MRS_Mstar_analysis/JWST_Mstar_dataAnalysis_analysis.ipynb

```diff
@@ -84,6 +84,7 @@
    "cell_type": "code",
```

Comment:
<p>Thanks, @rosteen, for your work on this notebook. Is there a way to dynamically assign a spatial_subset via API call(s) replicating the UI workflow? I see the execution fails for this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
This can be follow-up work. Thank you @haticekaratay for pointing it out. Can you please create the appropriate ticket so it is not forgotten?